tests/methods/aci-18/output.ref

```diff
@@ -0,0 +1,1147 @@
+  Cycle 1 took: 0.061820 s
+
+  ==> Cycle 2 <==
+
+  Initial P space dimension: 97
+  Spin-complete dimension of the P space: 118 determinants
+  Time spent building a_list   0.000625 s
+  Time spent building b_list   0.000621 s
+  Time spent building aa_list  0.000919 s
+  Time spent building bb_list  0.000894 s
+  Time spent building ab_list  0.002242 s
+  Time spent diagonalizing H:   0.007364 s
+
+    P-space  CI Energy Root   0        = -229.358528279106 Eh =   0.0000 eV
+
+  Using batched Q_space algorithm
+  Setting nbin to 1 based on estimated memory ( 0.302 MB)
+  WARNING: Batching not required, use another algorithm
+  Overwriting nbin to 2 based on user input
+  Number of bins for exitation space:  2
+  Number of threads: 1
+ F subspace (bin 0) takes 0.014015 s
+  Size of F subspace: 352 dets (bin 0)
+  Added 20 dets from bin 0, (td 0)
+ F subspace (bin 1) takes 0.010781 s
+  Size of F subspace: 406 dets (bin 1)
+  Added 20 dets from bin 1, (td 0)
+  Screened out 0.0007805958 Eh of correlation
+  Dimension of the truncated SD space: 40 determinants
+  Time spent building the external space (default): 0.026257 s
+
+  Time spent sorting: 0.000006
+  Time spent selecting: 0.000022
+  Dimension of the P + Q space: 142 determinants
+  Time spent screening the model space: 0.000043 s
+  Time spent building the model space: 0.026328
+  Spin-complete dimension of the PQ space: 232
+  Time spent building a_list   0.001191 s
+  Time spent building b_list   0.001198 s
+  Time spent building aa_list  0.001688 s
+  Time spent building bb_list  0.001688 s
+  Time spent building ab_list  0.004129 s
+  Total time spent diagonalizing H:   0.037862 s
+
+    PQ-space CI Energy Root   0        = -229.359450812288 Eh =   0.0000 eV
+    PQ-space CI Energy + EPT2 Root   0 = -229.360444943991 Eh =   0.0000 eV
+
+
+  Most important contributions to root   0:
+    0   0.813533 0.661836624          96 |02200202>
+    1  -0.490236 0.240331776          95 |22200200>
+    2  -0.112214 0.012591944          94 |00202202>
+    3  -0.084217 0.007092439          93 |02000222>
+    4   0.074859 0.005603927          92 |20202200>
+    5   0.066299 0.004395584          90 |0-+0-2+2>
+    6   0.066299 0.004395584          91 |0+-0+2-2>
+    7   0.065998 0.004355739          89 |22000220>
+    8  -0.063258 0.004001590          87 |-2-002++>
+    9  -0.063258 0.004001590          88 |+2+002-->
+
+  Spin state for root 0: S^2 = 0.000000, S = 0.000, singlet
+  Cycle 2 took: 0.090995 s
+
+  ==> Cycle 3 <==
+
+  Initial P space dimension: 97
+  Spin-complete dimension of the P space: 118 determinants
+  Time spent building a_list   0.000623 s
+  Time spent building b_list   0.000625 s
+  Time spent building aa_list  0.000898 s
+  Time spent building bb_list  0.000890 s
+  Time spent building ab_list  0.002240 s
+  Time spent diagonalizing H:   0.007410 s
+
+    P-space  CI Energy Root   0        = -229.358528279106 Eh =   0.0000 eV
+
+  Using batched Q_space algorithm
```

Comment:
It would be nice to improve the formatting here. Is there a way to print this out as a table?